canonical-web-and-design/ubuntu.com
# Smart Start moves to /smart-start/guide

## Done

- Move discourse content to /smart-start/guide according to [project brief](https://docs.google.com/document/d/1Rz_USCvgGL1b-kv0BKdE3GfvyK0NbSvBDdoQr04fPWo/edit)

## QA

- Open the demo and go to /smart-start/guide
- /smart-start/ should return a 404
